app/models/user.js

```diff
@@ -0,0 +1,41 @@
+const Sequelize = require('sequelize');
+
+exports.getModel = db => {
+  return db.define(
+    'user',
+    {
+      id: {
+        type: Sequelize.INTEGER,
+        autoIncrement: true,
+        allowNull: false,
+        primaryKey: true
+      },
+      firstname: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      lastname: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false,
+        unique: true,
+        validate: {
+          isEmail: true,
+          contains: '@wolox.com.ar'
```

Comment:
dont delegate this validation to the database , its ok to keep it here but also do it in the controller through a regular expression.